taca/nanopore/instrument_transfer.py

```diff
@@ -69,6 +75,50 @@ def main(args):
             final_sync_to_storage(run_path, rsync_dest, args.archive_dir, rsync_log)
 
 
+def delete_archived_runs(args):
+    logging.info("Finding locally archived runs...")
+    # Look for dirs matching run pattern inside archive dir
+    run_paths = [
+        path
+        for path in glob(os.path.join(args.archive_dir, "*", "*", "*"), recursive=True)
```

Comment:
Just being paranoid (since this is automatic rmdir after all): What's the behaviour of this if there are symlinks found which lead elsewhere?

Reply:
From https://docs.python.org/3/library/shutil.html

> rmtree.avoids_symlink_attacks
    Indicates whether the current platform and implementation provides a symlink attack resistant version of [rmtree()](https://docs.python.org/3/library/shutil.html#shutil.rmtree). Currently this is only true for platforms supporting fd-based directory access functions.
    Added in version 3.3.
    
Evaluates to true on prom